.github/workflows/check_labels.yml

```diff
@@ -0,0 +1,26 @@
+name: "Check Pull Request labels"
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+    branches:
+      - dev
+    types:
+      - labeled
+      - opened
+      - reopened
+      - synchronize
+      - unlabeled
```

Comment:
As we have a bot to adjust labels, are all these needed, would not just `labeled` and `unlabeled` be enough?